packages/core/src/Stores/Modules/Cashier/cashier-store.js

```diff
@@ -174,6 +175,11 @@ export default class CashierStore extends BaseStore {
         this.is_dp2p_visible = is_dp2p_visible;
     }
 
+    @action.bound
+    setP2pNotifications(notifications) {
+        this.p2p_notifications = notifications;
```

Comment:
if it's just a number, can we rename it to reflect that? like `this.p2p_notification_count = notification_count;`

Reply:
Please change the function name too.